messaging/src/test/java/org/axonframework/eventhandling/deadletter/DeadLetteredEventProcessingTaskTest.java

```diff
@@ -105,6 +112,29 @@ void taskProcessesLetterUnsuccessfullyWhenHandlersThrowsAnException() throws Exc
         verify(enqueuePolicy).decide(testLetter, testException);
     }
 
+    @Test
+    void useInterceptorToHandleError() throws Exception {
```

Comment:
I had to do a short double take to follow that the interceptor is causing the letter to not be enqueued again although it fails.
Just a thought, but perhaps add that switch in the test name?
Or, another idea that crossed my mind: an `AtomicBoolean` on the registered interceptor, to validate it being invoked.

Note, both aren't strictly necessary, as the test, in the end, is comprehensible and works. So, if you feel strongly about the current format, feel free to ignore my comments.